npm-shrinkwrap.json

```diff
@@ -2,6 +2,256 @@
   "name": "hof",
   "version": "6.0.0",
   "dependencies": {
```

Comment:
this is largely pointless until we publish a new version. Can you remove shrinkwrap from this PR as it's confusing things a little.